maquetado/scss/application.scss

```diff
@@ -0,0 +1,34 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+body {
+	font-family: 'Roboto', sans-serif;
+	margin: 0;
+	padding: 0;
+}
+
+.nav {
+
+}
+
+.nav {
+	height: 90px;
+	width: 100%;
+	position: fixed;
+	box-shadow: 0px 5px 10px $grey;
+
+	.nv-logo {
+		width: 15%;
+		float: left;
+		padding-top: 20px;
+		padding-left: 25px;
+	}
+
+	.nv-text {
+		font-size: 2em;
+		float: right;
+		padding-top: 20px;
```

Comment:
Center the elements in the navbar using flexbox as it will be easier if you need to change the navbar height in the future.